generators/client/templates/angular/src/main/webapp/app/core/language/_language.helper.ts

```diff
@@ -21,9 +21,9 @@ import { Title } from '@angular/platform-browser';
 import { Router, ActivatedRouteSnapshot } from '@angular/router';
 import { TranslateService, LangChangeEvent } from '@ngx-translate/core';
 
-import { LANGUAGES } from './language.constants';
+import { LANGUAGES } from '../../shared/language/language.constants';
```

Comment:
move the related files here as well, it doesnt make sense to have them in two places

Reply:
I didn't move the other files because in some cases, the files were component files, so if we want to re-use some component in other modules of the app, they should stay in shared. That's why I only moved the services into the `CoreModule`. But I agree for this constant, I'm moving it